README.md

```diff
@@ -90,13 +121,13 @@ var app = express();
 Describe the differences between Express and Rails as backend frameworks.
 
 ```text
-
+Rails is all about conventions; Express is more open-ended. Ruby vs JS.
 ```
 
 ### Question #8
 
-What is the importance of using body-parser in our express application for post requests? 
+What is the importance of using body-parser in our express application for post requests?
 
 ```js
-
+It lets you use information contained in HTML forms.
```

Comment:
:+1: we can also use it to process other inputs (e.g., JSON coming from an api).